kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerServiceNativeImpl.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2022 Eurotech and/or its affiliates and others
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Eurotech
+ ******************************************************************************/
+package org.eclipse.kura.ai.triton.server;
+
+import org.eclipse.kura.executor.CommandExecutorService;
+
+public class TritonServerServiceNativeImpl extends TritonServerServiceAbs {
+
+    @Override
+    TritonServerInstanceManager createInstanceManager(TritonServerServiceOptions options,
+            CommandExecutorService executorService, String decryptionFolderPath) {
+        return new TritonServerNativeManager(options, executorService, decryptionFolderPath);
+    }
+
+    @Override
+    boolean isConfigurationValid() {
+        return !isNullOrEmpty(this.options.getBackendsPath()) && !isNullOrEmpty(this.options.getModelRepositoryPath());
+    }
```

Comment:
Does it make sense to check also the ports and grpc call size (both required parameters)?

Reply:
Actually `TritonServerServiceOptions` is already performing the required checks on the ports [here](https://github.com/eclipse/kura/blob/66b343825e9a9f7c92a6353016e684cf2e669f38/kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerServiceOptions.java#L58-L76) and the GRPC max message dims [here](https://github.com/eclipse/kura/blob/66b343825e9a9f7c92a6353016e684cf2e669f38/kura/org.eclipse.kura.ai.triton.server/src/main/java/org/eclipse/kura/ai/triton/server/TritonServerServiceOptions.java#L93-L98). Shoudn't this already cover what we need?

Reply:
Yes, it is covered by that checks.